src/garage/torch/distributions/tanh_normal.py

```diff
@@ -0,0 +1,151 @@
+"""A Gaussian distribution with tanh transformation."""
+from collections import namedtuple
+
+import torch
+from torch.distributions import Distribution, MultivariateNormal
+
+
+class TanhNormal(Distribution):
+    # pylint: disable=abstract-method, arguments-differ
```

Comment:
which method did you not implement? why?

where do arguments differ?

please only disable something like arugments-differ directly at the violation line, not for the whole class

Reply:
1) I don't implement some functions that are properties of a distribution. They aren't necessary for implementation or debugging so I have found.

2) I disabled arguments differ for log_prob and for r_sample, because I need the option for different types of returns.

Reply:
One such function I don't implement is a function that calculates a cdf on the distribution.

Reply:
I've fixed 2) for now